### PR TITLE
Workbench:  integrate cardano-tracer

### PR DESCRIPTION
**Pull request by deepfire**
1. `cardano-node-service` -- a NixOS service
2. workbench integration -- with several profiles
3. a bunch of `Makefile` targets -- for the above profiles -- including the `-auto`, `-nix` and `-autonix` variants:
    - `startstop-tracer`
    - `10-tracer`
    - `forge-stress-tracer`
    - `chainsync-early-byron-tracer`
    - `chainsync-early-alonzo-tracer`

### CODE REVIEW

File: nix/nixos/cardano-tracer-service.nix [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          }
+          { epHost    = "127.0.0.1";
+            epPort    = 3101; ## supervisord.portShiftPrometheus
+          }
+        ];
+        ekgRequestFreq = 1;
+        hasPrometheus = {
+          epHost    = "127.0.0.1";
+          epPort    = 3200; ## supervisord.portShiftPrometheus
+        };
```
Comment (denisshevchenko) [inline nix/nixos/cardano-tracer-service.nix:43]: After RTView will be merged in `master` (https://github.com/input-output-hk/cardano-node/pull/3852), `hasRTView` will be added here.

**Review by denisshevchenko — APPROVED** [review]

**Comment by Jimbo4350** [comment]
bors cancel

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]
I asked in the node channel to not merge anymore PRs until we have a node RC. I expect to have one by the end of the day.

**Review by Jimbo4350 — APPROVED** [review]

**Comment by deepfire** [comment]
bors merge

**Pull request merged**